packages/featureflags/featureToggleManager.js

```diff
@@ -53,21 +53,23 @@ class FeatureToggleManager {
         }
     }
 
-    async isFeatureEnabled (keystoneContext, featureName, featuresContext) {
-        const context = get(keystoneContext, 'req', null)
-            ? keystoneContext
-            : { req: { features: await this.fetchFeatures() } }
-
-        const request = context.req
-        const headersFeatureFlags = get(request, ['headers', 'feature-flags'])
+    async isFeatureEnabled (context, featureName, featuresContext) {
```

Comment:
these changes would cause loading features for tests...
https://github.com/open-condo-software/condo/blob/3b5c737ff5f9d012b2c260068bf9b78e698505d4/packages/featureflags/featureToggleManager.js

Reply:
I want to have the same behavior for tests as in the production